components/script/dom/request.rs

```diff
@@ -444,8 +442,6 @@ impl Request {
         r.request.borrow_mut().body = input_body;
 
         // Step 41
```

Comment:
Should we remove this dangling comment and update the other step comments?

Reply:
ah, thanks for catching it. removing the dangling comments were missed to commit. but yes, let me update other step comments as well. 🙏

Reply:
Just updated them in da35d43.